### PR TITLE
#4822507: Implement the vertical divider and make the dividers configurable

### PR DESCRIPTION
**Pull request by veej**
Closes [#4822507](https://buildo.kaiten.io/4822507)

### tests performed
image

### CODE REVIEW

File: packages/bento-design-system/src/Divider/Divider.tsx [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
+import { bentoSprinkles, Box } from "../internal";
+import { DecorativeDividerConfig } from "./Config";
 import { divider } from "./Divider.css";
 
-export function Divider() {
-  const { separatorProps } = useSeparator({});
+type Props = {
+  orientation?: "horizontal" | "vertical";
+};
+export function Divider({ orientation = "horizontal" }: Props) {
```
Comment (veej) [inline packages/bento-design-system/src/Divider/Divider.tsx:10]: default is horizontal for retrocompatibility

File: packages/storybook/stories/index.tsx [hunk 2]
```diff
@@ -115,6 +116,11 @@ export const {
       custom: "customColor1",
     },
   },
+  decorativeDivider: {
+    height: 3,
```
Comment (Phyele) [inline packages/storybook/stories/index.tsx:120]:
```suggestion
    height: 2,
```

File: packages/bento-design-system/src/util/defaultConfigs.tsx [hunk 3]
```diff
@@ -147,6 +148,12 @@ export const chip: ChipConfig<typeof bentoSprinkles, string> = {
   radius: "circledX",
 };
 
+export const decorativeDivider: DecorativeDividerConfig<typeof bentoSprinkles> = {
+  height: 1,
```
Comment (Phyele) [inline packages/bento-design-system/src/util/defaultConfigs.tsx:152]:
```suggestion
  height: 2,
```

File: packages/bento-design-system/src/util/defaultConfigs.tsx [hunk 4]
```diff
@@ -147,6 +148,12 @@ export const chip: ChipConfig<typeof bentoSprinkles, string> = {
...
+  radius: 0,
```
Comment (Phyele) [inline packages/bento-design-system/src/util/defaultConfigs.tsx:153]:
```suggestion
  radius: "circledX",
```

**Review by Phyele — CHANGES_REQUESTED** [review]

**Review by Phyele — APPROVED** [review]

**Review by gabro — APPROVED** [review]

**Pull request merged**